packages/teams-js/test/private/privateAPIs.spec.ts

```diff
@@ -117,6 +117,10 @@ describe('AppSDK-privateAPIs', () => {
     'https://www.microsoft365.com',
     'https://tasks.office.com',
     'https://www.example.com',
+    'https://teams.cloud.microsoft',
+    'https://outlook.cloud.microsoft',
+    'https://m365.cloud.microsoft',
+    'https://anythingbecauseitsawildcard.cloud.microsoft',
```

Comment:
I like this, btw this is a duplicate of a previous PR. I will close that one though.

https://github.com/OfficeDev/microsoft-teams-library-js/pull/2355